domains/peacefullyzombie.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "ZombieRBX",
+           "email": "",
+           "discord": "1287108529357979709",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.n88GcCDFIJFz-YeCXwru9A-8ZKhWNEqZrqVpq1g09e_2fPD54X7k73eTpUum9pTwsF6tD742H2m-TDT_SWLHcHW7N4lXyCfbPojIHI78mGs0U5G-bS0Ug9ylvDw_Ov9-bBY6-LCmXCbQI_CN_iD_7tfCo7fPd2ZACrTlkfSX2yr5hnevOita0WbkGfyD_cZUNWDVucceM9dSBvhOYPw2nJbJZNfVtLx6uAIK8LnAdxJs377oKv-gVOZsB0Vp2O6jJJFbeiL-0jiyOWAkVX7_ozswFf_GJhM87G3oeh1uAPUUAVuulrjrrcNCDL2pZIBdFYcKb72Z7Gyyp8ydr2oSTA.D-fdSeaTDKPq_gJ30vGYbA.vw17Ak6ma4RULouRIVthrIEX6-46uX3mkdV07Lm_MhkAr3v_PK6u8uAtc8e8i0b_78q_G7gUo-xZKofjrn4QIofpkHpgD5imo8YpU_Vz7pw.FicBljDbswBWbxS32oPEvA"
+        },
+    
+        "record": {
+            "CNAME": "peacefullyzombie.is-a.dev"
```

Comment:
Invalid CNAME, it's the same record as the domain you are requesting which is invalid. Please see docs: https://docs.is-a.dev